syntax = "proto3";
package syft.core.auth;

import "proto/core/common/common_object.proto";
import "google/protobuf/empty.proto";

message SignedMessage {
  syft.core.common.UID msg_id = 1;
  string obj_type = 2;
  bytes signature = 3;
  bytes verify_key = 4;
  bytes message = 5;
}

message VerifyKey { bytes verify_key = 1; }
message VerifyAll { google.protobuf.Empty all = 1; }